dist/login.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset='utf-8'>
+    <title>Hootenanny</title>
+    <base href="/">
```

Comment:
I think this tag is breaking deployment under tomcat `/hootenanny-id2x`, because we can't assume all resources are at the root.  I'm doing a test deploy to see what else arises.